src/PostgREST/Error.hs

```diff
@@ -188,6 +188,10 @@ compressedRel Relationship{..} =
             "cardinality" .= ("many-to-one" :: Text)
           , "relationship" .= (cons <> " using " <> qiName relTable <> fmtEls (fst <$> relColumns) <> " and " <> qiName relForeignTable <> fmtEls (snd <$> relColumns))
           ]
+        O2O cons relColumns -> [
+            "cardinality" .= ("one-to-one" :: Text)
+          , "relationship" .= (cons <> " using " <> qiName relTable <> fmtEls (fst <$> relColumns) <> " and " <> qiName relForeignTable <> fmtEls (snd <$> relColumns))
```

Comment:
No idea how to appease codecov here. A disambiguation error usually could be triggered by defining an M2M plus M2O to the same table, but since it's a O2O, a junction table in between the tables won't define a M2M.

Reply:
Maybe this helps. I get a disambiguation error by doing this:

```sql
create table api.first (
  id int primary key,
  name text,
  second_id_1 int references api.second unique,
  second_id_2 int references api.second unique
);

create table api.second (
  id int primary key,
  name text
);
```

This query returns the error:

```http
GET /first?select=second(*)
```

Reply:
Awesome :100: . Also good thing that it can be disambiguated with `/first?select=second!second_id_1(*)`